arm-timeseriesinsights/2017-02-28-preview/Swagger/timeseriesinsights.json

```diff
@@ -0,0 +1,1780 @@
+          {
+            "$ref": "#/parameters/EnvironmentNameParameter"
+          },
+          {
+            "$ref": "#/parameters/EventSourceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The event source definition was successfully retrieved and is in the response.",
+            "schema": {
+              "$ref": "#/definitions/EventSourceResource"
+            }
+          },
+          "404": {
+            "description": "The subscription, resource group, environment, or event source could not be found."
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "EventSources"
+        ],
+        "operationId": "EventSources_Update",
+        "description": "Updates the event source with the specified name in the specified subscription, resource group, and environment.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/EnvironmentNameParameter"
+          },
+          {
+            "$ref": "#/parameters/EventSourceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "eventSourceUpdateParameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/EventSourceUpdateParameters"
+            },
+            "description": "Request object that contains the updated information for the event source."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The event source definition was successfully updated and is in the response.",
+            "schema": {
+              "$ref": "#/definitions/EventSourceResource"
+            }
+          },
+          "404": {
+            "description": "The subscription, resource group, environment, or event source could not be found."
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "EventSources"
+        ],
+        "operationId": "EventSources_Delete",
+        "description": "Deletes the event source with the specified name in the specified subscription, resource group, and environment",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/EnvironmentNameParameter"
+          },
+          {
+            "$ref": "#/parameters/EventSourceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
```

Comment:
what is difference between 200 and 204 here? seems to be the same from swagger model? When would you expect to send 200 v/s 204?